kubetest/azure.go

```diff
@@ -574,6 +574,33 @@ func (c Cluster) GetClusterCreated(clusterName string) (time.Time, error) {
 }
 
 func (c Cluster) TestSetup() error {
+
+	// Download repo-list that defines repositories for Windows test images.
+
+	downloadUrl, ok := os.LookupEnv("KUBE_TEST_REPO_LIST_DOWNLOAD_LOCATION")
+	if !ok {
+		// Env value for downloadUrl is not set, nothing to do
+		log.Printf("KUBE_TEST_REPO_LIST_DOWNLOAD_LOCATION not set. Using default test image repos.")
+		return nil
+	}
+
+	downloadPath := path.Join(os.Getenv("HOME"), "repo-list")
```

Comment:
suggestion: (feel free to ignore this is probably just for CI anyhow!) but dropping files in $HOME might be annoying to users if we ever tell people to use kubetest w/ azure on their local machines, maybe consider a file in temp or something

Reply:
Yeah, that makes sense. The changes I made were always with only CI in mind ( bad habbit, I know ). I have a patch further down the line to fix some small nits in kubetest, but want to see the CI going and working in the first.

Reply:
SGTM 👍